_data/simple-icons.json

```diff
@@ -363,7 +363,7 @@
         {
             "title": "Alipay",
             "hex": "00A1E9",
-            "source": "https://gw.alipayobjects.com/os/rmsportal/trUJZfSrlnRCcFgfZGjD.ai"
+            "source": "https://intl.alipay.com/"
```

Comment:
I just found https://global.alipay.com/docs/ac/website_hk/design which could be used as source and guidelines

Reply:
added